eng/create-baridtag.ps1

```diff
@@ -0,0 +1,24 @@
+Param(
+  [Parameter(Mandatory=$true)][string] $azdoToken,
+  [Parameter(Mandatory=$true)][string] $barToken,
+  [Parameter(Mandatory=$true)][string] $githubPAT
+)
+
+set-strictmode -version 2.0
+$ErrorActionPreference = 'Stop'
+
+. $PSScriptRoot\common\tools.ps1
+$darc = & "$PSScriptRoot\validation\get-darc.ps1"
+
+$global:azdoToken = $azdoToken
```

Comment:
Why are all the variables global?

Reply:
Agreed. I don't think you need the global: for anything you've got in this script

Reply:
Because I stole this script from another I wrote. I can unglobalize them since there's not much to this script as the one I stole it from.

Reply:
Yep, I think we can make them locals. globals make sense when you'll write lots of functions, or when you'll be using the script from another that may have conflicts.